HesoyamHospital/GraphicEditor/SearchService.cs

```diff
@@ -0,0 +1,88 @@
+﻿using Microsoft.EntityFrameworkCore.Internal;
+using System;
+using System.Collections.Generic;
+using System.Text;
+using System.Windows;
+
+namespace GraphicEditor
+{
+    class SearchService
+    {
+        public SearchService()
+        {
+        }
+
+        public List<MapLocation> Find_objects_by_name(string name) 
```

Comment:
This part is something that is considered as hardcoded as from what I understood.
The idea is to be able to dynamically create and load information about rooms.
As with the current code, that is not possible and you are bounded to these 5 files only.
Should be worked on in the near future.